ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -69,6 +69,24 @@ def _array_idxes_n_dtype(draw, **kwargs):
     return x, idxes, dtype
 
 
+@st.composite
+def _broadcastable_trio(draw):
+    shape = draw(helpers.get_shape(min_num_dims=1, min_dim_size=1))

```

Comment:
Each of the three input tensors need to be mutually broadcastable as you know, but setting the shape of each tensor to be the same is limiting the testing scope. There is a helper called `mutually_broadcastable_shapes` which you could use to generate three distinct but broadcastable shapes.

Reply:
@LucaLumetti Hi there, are you still working on this PR?

Reply:
@simonetgordon yes I am! Got a busy week but I'll go back on it in the next days

Reply:
@simonetgordon I've added the mutually_broadcastable_shapes as suggested and tried to fix the dtype problem. Now the first two tests of four are good. About the last two, when the backend is tensorflow and torch, I get an exception that I cannot understand: `ivy.exceptions.IvyBackendException: torch: is_float_dtype: torch: as_ivy_dtype: dtype('bool')` (same is with tensorflow).
The stacktrace is:
```
ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py:866: in test_torch_where
    helpers.test_frontend_function(
ivy_tests/test_ivy/helpers/function_testing.py:476: in test_frontend_function
    as_variable_flags = [
ivy_tests/test_ivy/helpers/function_testing.py:477: in <listcomp>
    v if ivy.is_float_dtype(d) and not test_flags.with_out else False
ivy/func_wrapper.py:128: in new_fn
    return fn(*new_args, **new_kwargs)
ivy/func_wrapper.py:459: in new_fn
    return fn(*args, **kwargs)
```
It looks like `ivy.is_float_dtype(d)` gets called on the type of the `condition` parameter, but I don't know when and why it works in half of the tests.